config/custom-environment-variables.json

```diff
@@ -98,5 +98,12 @@
       "encryptionKey": "COOKIE_ENCRYPTION_KEY",
       "AppInsightsInstrumentationKey":"APPINSIGHTS_INSTRUMENTATIONKEY"
     }
+  },
+
+  "launchDarkly": {
+    "sdk": "LD_SDK_KEYS",
+    "featureFlags": {
+       "admissions": "LD_FEATURE_ADMISSIONS"
```

Comment:
no need for this to be configurable per env, debatable on whether it should be in config at all

Reply:
that is true